airflow/gcp/example_dags/example_mlengine.py

```diff
@@ -0,0 +1,219 @@
+
+default_args = {
+    "start_date": airflow.utils.dates.days_ago(1),
+    "params": {
+        "model_name": MODEL_NAME
+    }
+}
+
+with models.DAG(
+    "example_gcp_mlengine", default_args=default_args, schedule_interval=None  # Override to match your needs
+) as dag:
+    training = MLEngineTrainingOperator(
+        task_id="training",
+        project_id=PROJECT_ID,
+        region="us-central1",
+        job_id="training-job-{{ ts_nodash }}-{{ params.model_name }}",
+        runtime_version="1.14",
+        python_version="3.5",
+        job_dir=JOB_DIR,
+        package_uris=[TRAINER_URI],
+        training_python_module=TRAINER_PY_MODULE,
+        training_args=[],
+    )
+
+    create_model = MLEngineModelOperator(
+        task_id="create-model",
+        project_id=PROJECT_ID,
+        operation='create',
+        model={
+            "name": MODEL_NAME,
+        },
+    )
+
+    get_model = MLEngineModelOperator(
+        task_id="get-model",
+        project_id=PROJECT_ID,
+        operation="get",
+        model={
+            "name": MODEL_NAME,
+        }
+    )
+
+    get_model_result = BashOperator(
+        bash_command="echo \"{{ task_instance.xcom_pull('get-model') }}\"",
+        task_id="get-model-result",
+    )
+
+    create_version = MLEngineVersionOperator(
+        task_id="create-version",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        operation='create',
+        version={
+            "name": "v1",
+            "description": "First-version",
+            "deployment_uri": '{}/keras_export/'.format(JOB_DIR),
+            "runtime_version": "1.14",
+            "machineType": "mls1-c1-m2",
+            "framework": "TENSORFLOW",
+            "pythonVersion": "3.5"
+        }
+    )
+
+    create_version_2 = MLEngineVersionOperator(
+        task_id="create-version-2",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        operation='create',
+        version={
+            "name": "v2",
+            "description": "Second version",
+            "deployment_uri": SAVED_MODEL_PATH,
+            "runtime_version": "1.14",
+            "machineType": "mls1-c1-m2",
+            "framework": "TENSORFLOW",
+            "pythonVersion": "3.5"
+        }
+    )
+
+    set_defaults_version = MLEngineVersionOperator(
+        task_id="set-default-version",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        operation='set_default',
+        version_name="v2",
+    )
+
+    list_version = MLEngineVersionOperator(
+        task_id="list-version",
+        project_id=PROJECT_ID,
+        model_name=MODEL_NAME,
+        operation='list',
+    )
+
+    list_version_result = BashOperator(
+        bash_command="echo \"{{ task_instance.xcom_pull('list-version') }}\"",
+        task_id="list-version-result",
+    )
+
+    prediction = MLEngineBatchPredictionOperator(
+        task_id="prediction",
+        project_id="polidea-airflow",
```

Comment:
PROJECT_ID ?

Reply:
Thanks. I am currently creating PR to improve it.

Reply:
https://github.com/apache/airflow/pull/6698